volim.css

```
@@ -333,4 +333,76 @@ body {
   font-size: 20px;
   margin-top: 10px;
   color: #252121;
-}
\ No newline at end of file
+}
+
+.shortcode{
+  display: block;
+  height: 700px;
+  width: 100%;
+}
+
+.shortcode-top{
+  display: block;
+  height: 17%;
+  width: 100%;
+}
+
+.shortcode-title{
+  display: block;
+  width: 300px;
+  height: 60px;
+  border-bottom: 1px solid black;
+  position: relative;
+  margin: auto;
+}
+
+.shortcode-title__text{
+  font-size: 45px;
+  font-family: 'Open Sans', sans-serif;
+  font-weight: 600;
+  text-align: center;
+}
+
+.shortcode-title__circle{
+ display: block;
+ width: 15px;
+ height: 15px;
+ position: absolute;
+ bottom: -8px;
+ right: 140px;
+ background: #b52f3a;
+ border-radius: 50%;
+}
+
+.shortcode-bottom{
+  display: block;
+  height: 77%;
+  width: 100%;
+}
+
+.shortcode-left{
```

Still a lot of classes without space before `{`
```suggestion
.shortcode-left {
```